protocol/avdl/chat1/common.avdl

```diff
@@ -209,6 +209,11 @@
     MessageID basis;
   }
 
+  @mpackkey("em") @jsonkey("em")
+  record ConvEphemeralMetadata {
+    gregor1.Time lastKtime;
```

Comment:
I don't see anywhere where we read this new field. Will that come later?

Reply:
yeah this is state to check if the server has purged messages on a conversation but isn't used until https://github.com/keybase/client/pull/11347